app/src/main/res/values-nl/help.xml

```diff
@@ -0,0 +1,166 @@
+        <p>Je wilt de portemonnee gebruiken die op je Ledger Nano S staat.</p>
+        <p>Je geheime sleutel wordt nooit opgehaald van je Ledger, dus je moet het apparaat iedere keer aansluiten als je deze portemonnee wilt gebruiken.</p>
+        <p>Voer een unieke naam en wachtwoord in. Met het wachtwoord worden de gegevens in je portemonnee op je Android-apparaat beveiligd. Gebruik een sterk wachtwoord - of liever een wachtzin van meerdere woorden.</p>
+        <p>Voer in het veld \"Herstelpunt\" het bloknummer in van de eerste transactie die voor dit adres is gebruikt. Je kunt ook een datum gebruiken, met de indeling JJJJ-MM-DD. Als je het niet zeker weet, kun je een schatting invoeren van de datum/blokhoogte <em>voordat</em> je het adres van deze portemonnee voor het eerst hebt gebruikt.</p>
+    ]]></string>
+
+    <string name="help_create_keys"><![CDATA[
+        <h1>Portemonnee maken - Sleutels</h1>
+        <p>Als je een Monero-adres herstelt met je sleutels.</p>
+        <p>Voer een unieke naam en wachtwoord in. Met het wachtwoord worden de gegevens in je portemonnee op dit apparaat beveiligd.
+        Gebruik een sterk wachtwoord - of liever een wachtzin van meerdere woorden.</p>
+        <p>Voer je Monero-adres in het veld \"Openbaar adres\" in, en vul de velden \"Alleen-lezen sleutel\" en \"Bestedingssleutel\" in.<p>
+        <p>Voer in het veld \"Herstelpunt\" het bloknummer in van de eerste transactie die voor dit adres is gebruikt. Je kunt ook een datum gebruiken, met de indeling JJJJ-MM-DD. Als je het niet zeker weet, kun je een schatting invoeren van de datum/blokhoogte <em>voordat</em> je het adres van deze portemonnee voor het eerst hebt gebruikt.</p>
+    ]]></string>
+
+    <string name="help_create_view"><![CDATA[
+        <h1>Portemonnee maken - Alleen-lezen</h1>
+        <p>Als je alleen de binnenkomende transacties voor een portemonnee wilt bekijken.</p>
+        <p>Voer een unieke naam en wachtwoord in. Met het wachtwoord worden de gegevens in je portemonnee op dit apparaat beveiligd.
+        Gebruik een sterk wachtwoord - of liever een wachtzin van meerdere woorden.</p>
+        <p>Voer je Monero-adres in het veld \"Openbaar adres\" in, en vul het veld \"Alleen-lezen sleutel\" in.<p>
+        <p>Voer in het veld \"Herstelpunt\" het bloknummer in van de eerste transactie die voor dit adres is gebruikt. Je kunt ook een datum gebruiken, met de indeling JJJJ-MM-DD. Als je het niet zeker weet, kun je een schatting invoeren van de datum/blokhoogte <em>voordat</em> je het adres van deze portemonnee voor het eerst hebt gebruikt.</p>
+    ]]></string>
+
+    <string name="help_details"><![CDATA[
+        <h1>Portemonneegegevens</h1>
+        <h2>Openbaar adres</h2>
+        Je openbare adres is te vergelijken met een bankrekeningnummer dat je aan iedereen kunt geven zonder het risico te lopen dat je je moneroj kwijtraakt. Anderen gebruiken dit adres om moneroj naar jouw portemonnee te sturen.
```

Comment:
The lowercased word "moneroj" is used, and in the English version "Monero" is used.

Reply:
Yeah, I wanted to be a bit more consistent than the source text, using 'Monero' for the network/protocol and 'monerujo' for the coins. Haven't done that everywhere, actually.